### ISSUE TITLE
Problem Reproducing ICM2O Results

### ISSUE THREAD

**Issue by tuliosouza99** [Leminhbinh0209/CVPR24-FAS#1]
### Description

First, congratulations on your great work! 

I need help reproducing the results described in this repository. I've followed the instructions provided, but I'm not getting the same outputs reported.

### Steps to Reproduce

1. Follow the preprocessing steps in [SAFAS](https://github.com/sunyiyou/SAFAS).
2. `python train.py --config ./configs/ICM2O.yaml` (Using the same ICM2O.yaml file provided on this repository)

### Expected Results

HTER=0.086, AUC=0.9716 (as reported on [3.3 Snapshot Results](https://github.com/Leminhbinh0209/CVPR24-FAS?tab=readme-ov-file#33-snapshot-resutls))

### Actual Results

[Best result]: epoch=140, HTER=0.1133, AUC=0.9605

Generated [log.txt](https://github.com/Leminhbinh0209/CVPR24-FAS/files/14558777/log.txt), [summary.txt](https://github.com/Leminhbinh0209/CVPR24-FAS/files/14558782/summary.txt) and [epoch_140_OULU-NPU_score.txt](https://github.com/Leminhbinh0209/CVPR24-FAS/files/14559099/epoch_140_OULU-NPU_score.txt) files are provided.

### Additional Context

It may be related to a difference in the dataset size. I've checked your supplementary material, and you report that OULU has 2960 spoof videos and 990 live videos. Although I also have 990 live videos, mine consists of 3960 spoof videos. The other datasets (CASIA, Replay, and MSU) match in size.

Any guidance or insights would be greatly appreciated. Thank you for your time!

**Comment by Leminhbinh0209** [comment]
Hi,

Thanks for your interest in our work. 
I was a typo of "2,960" and your actual number is correct. About the results, I think you may need to try with different random seed (`line 124`) in `train.py`, and with augmentations (`transforms.ColorJitter`) mentioned in the [bag of tricks.](https://github.com/Leminhbinh0209/CVPR24-FAS/blob/main/README.md#32-bag-of-tricks) In my experiment, many FAS methods rely on random seeds, including mine. So, hope that you obtain expected results.

**Comment by tuliosouza99** [comment]
Thanks! I was able to reduce the HTER to 10.00% but was still unable to achieve 8.60 +-0.28 from the results section. I consulted your released weights and noticed these hyperparams: 

![hyperparams](https://github.com/Leminhbinh0209/CVPR24-FAS/assets/49206513/98f13750-3f01-462c-8662-b730c0f9d780)

I presume MDSAM was renamed to GAC-FAS in the final release, but any chance "ASAM" and "AWP" were used in any part of the code?
